File: DonatorSange-Angular/src/app/models/donorData.ts
import { Donor } from "./donor";
import {BloodType } from "./bloodType";

export class DonorData{
    id:number;
    birthdate:Date;
    bloodType:BloodType;
    weight:number;
    heartBeat:number;
    bloodPressure:number;
    interventions:boolean;
    feminineProblems:boolean;
    junkFood:boolean;
    onDrugs:boolean;
    diseases:boolean;
}

File: AntiClimacus/DataAccessLayer/Entities/DonorData.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DataAccessLayer.Entities
{
    public class DonorData
    {
        public int Id { get; set; }
        public int DonorId { get; set; }
        public DateTime Birthdate { get; set; }
        public int Weight { get; set; }
        public int BloodTypeId { get; set; }
        public int Heartbeat { get; set; }
        public int BloodPressure { get; set; }
        public bool Interventions { get; set; }
        public bool FeminineProblems { get; set; }
        public bool JunkFood { get; set; }
        public bool OnDrugs { get; set; }
        public bool Diseases { get; set; }

        public BloodType BloodType { get; set; }
        public Donor Donor { get; set; }
    }
}
